app/components/listing/feedback/FeedbackModal.jsx

```diff
@@ -0,0 +1,169 @@
+import React, { Fragment, useState } from 'react';
+
+import {
+  FeedbackTags, Review, SubmitMessage, tagList,
+} from './FeedbackSteps';
+
+import { images } from '../../../assets';
+import './FeedbackModal.scss';
+import { addFeedback } from '../../../utils/DataService';
+
+const getUrl = ([path, id]) => (
+  `/api/${path === 'organizations' ? 'resources' : path}/${id}/feedbacks`
+);
+
+const FeedbackModal = ({ closeModal }) => {
+  const [upvote, setUpvote] = useState(false);
+  const [downvote, setDownvote] = useState(false);
+  const [tagOptions, setTags] = useState(tagList);
+  const [review, setReview] = useState('');
+  const [step, setStep] = useState(0);
+  const [reviewRequired, setReviewRequired] = useState(false);
```

Comment:
Can I write helper function like this 
``` js
const isReviewRequired = () => (
    tagOptions.some(({ tag, selected }) => tag === 'Other' && selected)
);
```
and invoke it when I need it's value?
Example:
``` js       
<button
    type="button"
    className="button"
    disabled={isReviewRequired() && vote === downvote}
    onClick={handleSubmit}
>
    Submit
</button>
```

Reply:
I think that's close, but I don't think you actually need to turn it into a function because it's not expensive to compute. It's clearer to just unconditionally do the computation and assign the value to a local variable than to hide it behind a function definition with a closure. This is part of the React way of doing things, where you view functions as being pure (side-effect-free) transformations on state (which includes props).

Also, I would also pull out the `vote === downvote` part into your `isReviewRequired` definition, since, in my opinion, that is part of the business logic for determining whether you would allow someone to submit the form.